.github/workflows/deploy-integration-tests.yaml

```diff
@@ -16,7 +16,7 @@ on:  # yamllint disable-line rule:truthy
         type: choice
         options:
           - both
-          - providers-integration-tests
+          - providers-integration-tests-on-CE
```

Comment:
The reason I name it `providers-integration-tests` and `providers-integration-tests-on-KE` is because they're the deployment names. If we're to change it, maybe we should change the name on deployment as well.

Reply:
okay, removed the rename. We can address it later.